destreamer.ts

```diff
@@ -77,34 +77,28 @@ function init() {
         console.info(colors.blue("There will be no video downloaded, it's only a simulation\n"));
 }
 
-async function DoInteractiveLogin(username?: string): Promise<Session> {
+async function DoInteractiveLogin(url: string, username?: string): Promise<Session> {
+
+    let videoID = url.split("/").pop() ?? process.exit(25);

```

Comment:
Let's print what error code 25 means to the console before .exit().

Reply:
I mean.. if you want to we could have a struct in `Types.ts` that holds the mapping between error codes and description.

Reply:
I have already found a better way, there's proces.on('exit', ..blabla...)that is called whenever we exit and there we can use the exit code to print an error based on what code it is, I'll implement it in some future PR

For now I'll do a temporary fix

Reply:
Right, this is still Node we can .on() stuff, yes let's do that, i think it will be very neat. Merging as is, open a new PR for the beautified error handling.